spec/controllers/users/checkins_controller_spec.rb

```diff
@@ -118,6 +118,16 @@
       delete :destroy_all, params: other_user_params
       expect(response).to redirect_to(root_path)
     end
+
+    context "with date params" do
+      before { FactoryGirl.create(:checkin, device: device, created_at: 1.day.ago) }
+
+      it "deletes all checkins within date range specified" do
+        expect {
+          delete :destroy_all, params: params.merge(from: 1.day.ago, to: 1.day.ago)
```

Comment:
The test doesn't really cover a range of dates. Just a day. Should hardcode some assertions.